README.md

```diff
@@ -109,6 +109,8 @@ nexusIQScan {
     allConfigurations = false // if true includes the dependencies in all resolvable configurations. By default is false, meaning only 'compileClasspath', 'runtimeClasspath', 'releaseCompileClasspath' and 'releaseRuntimeClasspath' are considered
     resultFilePath = 'results.json' // Optional. JSON file containing results of the evaluation
     modulesExcluded = ['module-1', 'module-2'] // Optional. For multi-module projects, the names of the sub-modules to exclude from scanning and evaluation.
+    dirIncludes = 'some-ant-pattern' // Optional. Ant-like glob patterns to select directories/archives that should be examined
+    dirExcludes = 'some-ant-pattern' // Optional. Ant-like glob patterns to select directories/archives that should be excluded. For Android we suggest using '**/classes.jar,**/annotations.zip,**/lint.jar'
```

Comment:
Suggestion. Instead of `some-ant-pattern` maybe give an example? e.g. `**/classes.jar`. 

Also may state it can be comma separated e.g. "Comma separated list of Ant-like patterns ..."

Reply:
> Suggestion. Instead of `some-ant-pattern` maybe give an example? e.g. `**/classes.jar`.
> Also may state it can be comma separated e.g. "Comma separated list of Ant-like patterns ..."

Although I added example patterns for Android projects in the comment, I hesitate adding examples directly in the value as I've seen some users copy-paste the whole configuration section when reviewing sample projects attached in the issues they raise; so I prefer using a value that's is clearly an example rather than something that can be interpreted as a valid default value.

1885f59e0595ffe731249ad188df05b66cfb9697